cli.js

```diff
@@ -0,0 +1,54 @@
+const { validate, validateStats} = require('./validate');
+const { getLinks, uniqueLinks,stats} = require('./stats');
+const fs = require("fs")
+const path = require('path');
+const marked = require('marked');
+//const path = require('path');
+
+const opts={
+    file:{
+        demand:true,
+        alias: "f"
+        }
+}
+
+const argv = require("yargs")
+                .command("validate","imprime en consola",opts)
+                .command("stats","imprime en consola",opts)
+                .command("validateStats","imprime en consola",opts)
+                .argv
+
+
+let data = fs.readFileSync(`${argv.file }`,"utf-8");
+const links = getLinks(data)
```

Comment:
let se utiliza cuando los valores mutan y const cuando lo valores no cambian
```suggestion
let links = getLinks(data)
```